.github/workflows/crowdin.yml

```diff
@@ -75,7 +74,7 @@ jobs:
           # abort if the pr already exists
           apt update && apt install -y gh
           PR_EXISTS=$(gh pr list --base "$BASE_BRANCH" --head "$HEAD_BRANCH" --state open --json number --jq '.[0].number')
-          if [ -z "$PR_EXISTS" ]; then
+          if [ -n "$PR_EXISTS" ]; then
```

Comment:
`-n` is true if `PR_EXISTS` is not an empty string. I can never remember shell scripting 😢